crates/miden-objects/src/note/note_tag.rs

```diff
@@ -118,7 +118,12 @@ impl NoteTag {
     /// The exponent of the maximum allowed use case id. In other words, 2^exponent is the maximum
     /// allowed use case id.
     pub(crate) const MAX_USE_CASE_ID_EXPONENT: u8 = 14;
-
+    // Creating a tag for local execution directly from AccountId defaults to using 14 bits.
+    pub const DEFAULT_LOCAL_TAG_LENGTH: u8 = 14;
+    // Creating a tag for network accounts should always use 30 bits.
+    pub const DEFAULT_NETWORK_TAG_LENGTH: u8 = 30;
+    /// The maximum number of bits that can be encoded into the tag for local accounts.
+    pub const MAX_LOCAL_TAG_LENGTH: u8 = 30;
```

Comment:
```suggestion
    /// The default note tag length for an account ID with local execution.
    pub const DEFAULT_LOCAL_TAG_LENGTH: u8 = 14;
    /// The default note tag length for an account ID with network execution.
    pub const DEFAULT_NETWORK_TAG_LENGTH: u8 = 30;
    /// The maximum number of bits that can be encoded into the tag for local accounts.
    pub const MAX_LOCAL_TAG_LENGTH: u8 = 30;
```

Should we make those doc comments?

Reply:
yes I missed that, thanks